src/components/application_manager/src/app_launch/app_launch_data_json.cc

```diff
@@ -111,12 +114,22 @@ Json::Value& AppLaunchDataJson::GetApplicationListAndIndex(
   return apps_list;
 }
 
+Json::Value& AppLaunchDataJson::GetApplicationListAndIndex(
+    const ApplicationData& app_data, int32_t& founded_index) const {
+  static Json::Value value;
+  return value;
```

Comment:
@LitvinenkoIra Do we still need this override when we have 

`Json::Value& AppLaunchDataJson::GetApplicationListAndIndex(
    const ApplicationData& app_data,
    int32_t& founded_index,
Json::Value& dictionary) const`
?

Reply:
@ZhdanovP removed